tds-dll-mysql/src/main/java/tds/dll/mysql/StudentDLL.java

```diff
@@ -7121,6 +7121,23 @@ public SingleDataResultSet execute (SQLConnection connection) throws ReturnStatu
     return null;
   }
 
+  public boolean _SecureBrowserRequired_FN (SQLConnection connection, String clientname) throws ReturnStatusException {
```

Comment:
Why did you choose to add this within the TDS infrastructure rather than the microservices?  Is it because of how the loader system pulls all this data?

We have a microservice endpoint that fetches client system flags already in the config service that probably can be modified to get this data.

Reply:
I'll look at the config coming from the microservices and see if it fits better before I merge this pr.